### ISSUE TITLE
False positive when a class have suppressed MISSING_TO_STRING_OVERRIDE

### ISSUE THREAD

**Issue by facudev** [Monits/findbugs-plugin#4]
When a class have suppressed MISSING_TO_STRING_OVERRIDE and this class is used in another, FindBugs reports MISSING_FIELD_IN_TO_STRING.

Ex: 

```java
@SuppressFBWarnings(value = 'MISSING_TO_STRING_OVERRIDE',
		justification = 'We don\'t want to implement toString() to show the breed')
public class dog {
    private String breed;
}

public class MyDog {
    private Dog dog;
    private String name;

    @Override
	public String toString() {
		return 'MyDog{'
			+ 'name=' + name
			+ '}';
	}
}
```

When analyzing MyDog it says:
`The field dog is not included in the current toString implementation.
  Having a meaningful and complete toString will help developers debug code, by providing more meaningful
  inspections out of the box. For more information, refer to Effective Java, item 10.`
